wabt.js

```diff
@@ -3,11 +3,11 @@ const fs = require('fs')
 const cp = require('child_process')
 
 exports.compile = function (wast) {
-  return util.promisify(fs.writeFile)(`${__dirname}/temp.wast`, wast)
+  return util.promisify(fs.writeFile)(`${__dirname}/tmp/${testName}.wast`, wast)
```

Comment:
The point of the `tmp` directory is only for easy inspection?

Reply:
right